src/sdg_hub/prompts.py

```diff
@@ -41,3 +42,62 @@ def nemotron_chat_template():
 {%- if add_generation_prompt %}
   {{- '<|start_header_id|>assistant<|end_header_id|>\n\n' }}
 {%- endif %}"""
+
+
+### Granite 3.3 2B Chat Template
+@PromptRegistry.register("ibm-granite/granite-3.3-2b-instruct")
```

Comment:
```suggestion
@PromptRegistry.register("ibm-granite/granite-3.3")
```

If the chat templates for 2b and 8b are the same, could we change this to be common triggered for both the models.